index.html

```diff
@@ -61,5 +61,194 @@ <h1 class="hero-intro">I’m Akorede. <br> Glad to see <br class="see-break"> yo
             </ul>
         </div>
     </header>
+    <main>
+        <section class="project-section">
+            <article class="project-container">
+                <div>
+                    <a href="">
+                        <img src="images/tonic.png" alt="">
+                    </a>
+                </div>
+                <div class="project-description">
+                    <h2>Tonic</h2>
+                    <ul class="experience-list">
+                        <li>CANOPY</li>
+                        <li>Backend Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                    <ul class="skill-list">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button>
+                        See Project
+                    </button>
+                </div>
+            </article>
+            <article  class="project-container">
+                <div>
+                    <a href="">
+                        <img src="images/multi-stories.png" alt="">
+                    </a>
+                </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, please add padding/margin or space to the right side so that the project cards do not hide as seen in the image below. 
- [ ] Your `See project` button does not look like that of the Figma design. See the image below and fix it. 

![image](https://github.com/Akorede3133/portfolio/assets/84156714/8cabc6c0-bd77-46e5-ba84-9bf74e671eb4)